src/common/components/repository-input/index.js

```diff
@@ -17,27 +17,29 @@ export class RepositoryInput extends Component {
     const input = this.props.repositoryInput;
     let message;
 
-    if (input.repository && input.isFetching) {
+    if (input.inputValue.length > 2 && !input.repository) {
```

Comment:
minor ux nit, too early reporting of "invalid" input is annoying, is 2 the smallest this can be?